app/src/main/shared/actions/AuthActions.ts

```diff
@@ -43,6 +44,10 @@ class AuthActions extends AbstractActions implements Actions {
     logOut(): boolean {
         return true;
     }
+
+    sendPasswordResetEmail() : void {
+
+    }
```

Comment:
Don't actions needs to return something (e.g. `true`) in order for Alt to dispatch them?